plugins/node/opentelemetry-instrumentation-aws-sdk/test/lambda.test.ts

```diff
@@ -14,12 +14,8 @@
  * limitations under the License.
  */
 
-import { AwsInstrumentation } from '../src';
-import {
-  getTestSpans,
-  registerInstrumentationTesting,
-} from '@opentelemetry/contrib-test-utils';
-registerInstrumentationTesting(new AwsInstrumentation());
+import { getTestSpans } from '@opentelemetry/contrib-test-utils';
```

Comment:
This import from contrib-test-utils doesn't absolutely need to be before `./load-instrumentation` does it?